docs/README.md

```diff
@@ -2,3 +2,137 @@
 
 ![Architecture](./Architecture.png)
 
+
+
+## API
+
+### Network 
```

Comment:
**suggestion**: how about change the API docs format?

I suggest following format: https://gist.github.com/iros/3426278

**suggestion**: change API header from h2(##) to h1(##), and network from h3(###) to h2(##)

Reply:
That sounds good!